src/Compilers/CSharp/Portable/Symbols/Synthesized/SynthesizedParameterSymbol.cs

```diff
@@ -18,14 +18,14 @@ namespace Microsoft.CodeAnalysis.CSharp.Symbols
     /// </summary>
     internal abstract class SynthesizedParameterSymbolBase : ParameterSymbol
     {
-        private readonly MethodSymbol _container;
+        private readonly MethodSymbol? _container;
         private readonly TypeWithAnnotations _type;
         private readonly int _ordinal;
         private readonly string _name;
         private readonly RefKind _refKind;
 
         public SynthesizedParameterSymbolBase(
-            MethodSymbol container,
+            MethodSymbol? container,
```

Comment:
I couldn't spot how we could get a `null` in here. Can you confirm that's possible?

Reply:
It is used to produce parameter symbols for lowering expression lambdas.  See `ExpressionLambdaRewriter.cs` line 802.

---
In reply to: [386597911](https://github.com/dotnet/roslyn/pull/42068#discussion_r386597911) [](ancestors = 386597911)